### PR TITLE
use label instead of font color

### PR DESCRIPTION
**Pull request by fabianfrz**
do some code optimisation and use label instead of font color. It looks like this now:
screenshot_20180127_125712

### CODE REVIEW

**Comment by marjohn56** [comment]
Nice one... I just mentioned that to Ad, literally seconds before your message came in!  👍

File: src/www/widgets/widgets/gateways.widget.php [hunk 1]
```diff
@@ -34,45 +34,41 @@ function gateways_widget_update(sender, data)
...
             case 'none':
-              status_color = 'green';
-              break;
-            default:
-              status_color = '';
+              status_color = 'success';
               break;
           }
-          $("#"+tr_id+" > td:eq(3) > font").removeAttr('color');
+          console.log($("#"+tr_id+" > td:eq(3) > span"))
```
Comment (fichtner) [inline src/www/widgets/widgets/gateways.widget.php]: zap

File: src/www/widgets/widgets/gateways.widget.php [hunk 2]
```diff
@@ -34,45 +34,41 @@ function gateways_widget_update(sender, data)
...
+          $("#"+tr_id+" > td:eq(3) > span").removeClass("label-danger label-warning label-success label");
+          console.log(status_color);
```
Comment (fichtner) [inline src/www/widgets/widgets/gateways.widget.php]: zap! :)

**Comment by marjohn56** [comment]
It might be even better to specify the gateway label colours in the _variables.scss file.

**Comment by fichtner** [comment]
I think this is ok, they should be adjusted to warning/success/danger. Some other widgets made a similar transition.

**Comment by fabianfrz** [comment]
@fichtner did your requested changes - thanks for finding it - I used it for debugging because firefox did not respect my breakpoints for some reasons.

**Comment by fichtner** [comment]
Sure, that's what review are for. Thanks, merged.

**Pull request merged**